spec/models/revision_spec.rb

```diff
@@ -57,11 +58,21 @@
                features: {
                  refs_tags_key => 10
                })
+        create(:revision,
+               mw_rev_id: mw_rev_id + 1,
+               article_id: 90010260,
+               mw_page_id: 90010260,
+               new_article: true,
+               features: {
+                 wiki_refs_tags_key => 10
```

Comment:
This is testing the Wikidata version, but the context says Wikipedia. This may lead to confusion later on.

I recommend being a little more explicit, either assigning a name or or using `lets` to make it clear which is a Wikidata revision and which is a Wikipedia revision. I also think this is more likely to be a stable test if it's done with a Wikidata example fully — including using a real mw_page_id from Wikidata, and setting the wiki for the revision to be Wikidata as well.

Reply:
Had to use VCR, as was throwing `VCR::Errors::UnhandledHTTPRequestError:`
Also using VCR in just `it` block where revisions were created in `before` block was throwing the same error